### PR TITLE
Capstone spec

### PR DESCRIPTION
**Pull request by Colton-V-Kaneria**
Rough draft of Capstone Project Specs, with links and images transferred.

### CODE REVIEW

File: capstone.html [hunk 1]
```diff
@@ -0,0 +1,1062 @@
...
+						<a class="nav-link" href="#pj-heading-schematics">Schematics</a>
+					</li>
+					<li class="nav-item">
+						<a class="nav-link" href="#pj-heading-arduino-libraries">Installing Arduino Libraries</a>
+					</li>
+					<li class="nav-item">
+						<a class="nav-link" href="#pj-heading-installing-kicad">Installing KiCAD 7.0</a>
+					</li>
+					<li class="nav-item">
+						<a class="nav-link" href="#pj-analog-joystick">Analog Joystick</a>
```
Comment (dylanvu) [inline capstone.html:61]: clicking on this does not work, I believe that pj-analog-joy id does not exist anywhere in the document

File: capstone.html [hunk 2]
```diff
@@ -0,0 +1,1062 @@
...
+				<br><br>
+				<b>Install the newest version</b> of the NRF24L01_W library as well.
+				<br><br>
+				As a final suggestion, look back at your old project submissions and reuse applicable code.
+			</p>
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-references">References</h2>
+			<ul>
+				<li>
+					<a class="link-primary"
```
Comment (dylanvu) [inline capstone.html:976]: imo for all of these links, I would open them in a new tab

This is done by adding:

`target="_blank" rel="noopener noreferrer"` to the anchor tag.

If you're confused, google "html open anchor in new tab"

File: capstone.html [hunk 3]
```diff
@@ -0,0 +1,1062 @@
...
+				<tr>
+					<td><span class="text-red">+5V</span></td>
+					<td>5V</td>
+				</tr>
+				<tr>
+					<td><span class="text-blue">VX</span></td>
+					<td>A0</td>
+				</tr>
+				<tr>
+					<td><span class="text-pink">VY</span></td>
```
Comment (dylanvu) [inline capstone.html:380]: I believe the pink color doesn't work for me, how did you get the pink to work?

File: capstone.html [hunk 4]
```diff
@@ -0,0 +1,1062 @@
...
+				
+			</p>
+			<img class="img-fluid my-3 me-lg-5 me-3 float-start" style="max-width: 400px;"
+				src="assets/images/capstone/ops_joystick_pinout.png">
+			<h4 class="mb-3 fw-light">Joystick Pinout</h4>
+			<p class="mb-5">
+				To the left is the pinout of the joystick. The GND and <span class="text-red">+5V</span> pins power the radio module. 
+				Make sure they are connected appropriately to the Arduino board.
+				<br><br>
+				The next two pins are <span class="text-blue">VX</span> and <span class="text-pink">VY</span>, which carry analog signals for
```
Comment (dylanvu) [inline capstone.html:349]: dunno if the pink color works for me

File: capstone.html [hunk 5]
```diff
@@ -0,0 +1,1062 @@
...
+			<p class="mb-5">
+				We will use an L293D chip to drive the two DC motors of the rover. The H-bridge should be familiar to you. This time, the L293D will do 
+				the heavy lifting. Within this IC are two H-bridges, making it possible to drive two motors bidirectionally.
+			</p>
+
+			<h4 class="mb-3 fw-light">Reading the Big Scary Manual</h4>
+			<p class="mb-3">
+				This time around, we will not thoroughly discuss the driver’s operation. You will have to read the datasheet, and figure it out yourself.
+				<br><br>
+				<a class="link-primary" href="https://drive.google.com/file/d/1XgJbA4hapVwiDeIkIh0e_JCzlX8x-nO8/view">L293D Quadruple Half-H Driver Datasheet.pdf</a>
```
Comment (dylanvu) [inline capstone.html:438]: I believe that we want these PDFs to be actually downloaded and linked directly.

Check out `/assets/datasheets`

File: capstone.html [hunk 6]
```diff
@@ -0,0 +1,1062 @@
...
+								any Output Power pins.</b>
+							</li>
+						</ul>
+						<li>
+							Make sure every symbol is assigned to the correct footprint.
+						</li>
+					</ul>
+				</li>
+				<li class="list-group-item">
+					Download this <a class="link-primary"
```
Comment (dylanvu) [inline capstone.html:777]: Given the conventions that we have used to create new pages (basically, anything we host on google drive should actually be hosted locally), I believe that we would want to keep this file locally to the website.  I would probably actually create a new folder under `assets` that is relevant to this file, and put the file in here.

File: capstone.html [hunk 7]
```diff
@@ -0,0 +1,1062 @@
...
+					<a class="link-primary" 
+					href="https://drive.google.com/file/d/1cz1hD7DHxZB3xJh50DLlUIqBDYxhczFi/view">Arduino Nano Datasheet.pdf</a>
+				</li>
+				<li>
+					<a class="link-primary" 
+					href="https://drive.google.com/file/d/1MVRedo0v8xRIBDfJDWlC6YpZi6wRghAx/view">Resistor Color Code Chart.pdf</a>
+				</li>
+				<li>
+					<a class="link-primary" 
+					href="https://drive.google.com/file/d/1NOQYAHhi2zEvs3LF03NmtVruFnfNxvSp/view">NRF245L01_W Library</a>
```
Comment (dylanvu) [inline capstone.html:993]: This library should be downloaded and linked to `assets/libraries`

File: capstone.html [hunk 8]
```diff
@@ -0,0 +1,1062 @@
...
+				<b>Install the newest version</b> of the NRF24L01_W library as well.
+				<br><br>
+				As a final suggestion, look back at your old project submissions and reuse applicable code.
+			</p>
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-references">References</h2>
+			<ul>
+				<li>
+					<a class="link-primary" 
+					href="https://drive.google.com/file/d/1w9haYGk6eka_TOTiymXVfK3o4RiIGU-8/view">NRF24L01 Datasheet.pdf</a>
```
Comment (dylanvu) [inline capstone.html:977]: for all these google drive links, we would actually put these on our website under the `assets` and link to that.

**Review by dylanvu — CHANGES_REQUESTED** [review]

File: capstone.html (hunk 1 above)
Comment (Colton-V-Kaneria) [reply capstone.html:61]: My bad, I realized I left out "heading" in some pj link references.

File: capstone.html (hunk 2 above)
Comment (Colton-V-Kaneria) [reply capstone.html:976]: Should these pdfs be downloaded on click like the others or still just be site links in another tab?

File: capstone.html (hunk 5 above)
Comment (Colton-V-Kaneria) [reply capstone.html:438]: Fixed

File: capstone.html (hunk 6 above)
Comment (Colton-V-Kaneria) [reply capstone.html:777]: Fixed

File: capstone.html (hunk 7 above)
Comment (Colton-V-Kaneria) [reply capstone.html]: Fixed

File: capstone.html (hunk 8 above)
Comment (Colton-V-Kaneria) [reply capstone.html]: I'm a little confused by this. Is the link supposed to access the assets/datasheets folder and open a new tab for the file it accesses?

File: capstone.html (hunk 3 above)
Comment (dylanvu) [reply capstone.html:380]: works now

File: capstone.html (hunk 1 above)
Comment (dylanvu) [reply capstone.html]: looks good

File: capstone.html (hunk 2 above)
Comment (dylanvu) [reply capstone.html:976]: pdfs should open up to pdfs in a new tab also

File: capstone.html (hunk 6 above)
Comment (dylanvu) [reply capstone.html:777]: I still see google drive links

File: capstone.html (hunk 7 above)
Comment (dylanvu) [reply capstone.html]: looks good

File: capstone.html (hunk 5 above)
Comment (dylanvu) [reply capstone.html]: looks good

File: capstone.html (hunk 6 above)
Comment (dylanvu) [reply capstone.html:777]: they should be modified to be similar to the edit you made to the quadruple half h driver pdf file

File: capstone.html (hunk 8 above)
Comment (dylanvu) [reply capstone.html]: see other comments about the pdfs

**Review by dylanvu — APPROVED** [review]

**Pull request merged**